### ISSUE TITLE
Rake 11.2.0 missing test:recent task for Rails 2.3 and 3.2 projects

### ISSUE THREAD

**Issue by jasnow** [ruby/rake#139]
For all Rails 2.3 and 3.2 Rails projects:

When I run "rake test:recent" for "rake, version 11.1.2", it works and I get this:
```
rake test:recent     # Run tests for {:recent=>"test:prepare"} / Test recent changes
```
After upgrading to "rake, version 11.2.0", when I run "rake test:recent", I get this:
```
rake test:recent
rake aborted!
Don't know how to build task 'test:recent' (see --tasks)
```
Here is the stack trace: https://gist.github.com/jasnow/8d85484b04fc7864f408b3b7e881365a

**Comment by hsbt** [comment]
@jasnow Thank you for your report. It's regression between 11.1.2 and 11.2.0. I'm going to release 11.2.1.

**Comment by hsbt** [comment]
I released rake 11.2.2. (I missed to add another defect at 11.2.1 :( )

And I confirmed to work `test:recent` with Rails 3.2.x and Rake 11.2.2.

**Comment by jasnow** [comment]
THANKS !!!!
